apps/opik-backend/src/main/java/com/comet/opik/domain/ExperimentDAO.java

```diff
@@ -1151,4 +1152,34 @@ private Publisher<ExperimentGroupAggregationItem> mapExperimentGroupAggregationI
                     .build();
         });
     }
+
+    private static final String UPDATE = """
+            ALTER TABLE experiments UPDATE
```

Comment:
`UPDATE` queries are not performant in clickhouse. This should be an `INSERT` which will be eventually squashed by clickhouse.
Some changes are required to preserve the `PATCH` logic, i.e. read the existing experiment and only update non-null fields